Src/Newtonsoft.Json.UnityConverters/Math/SphericalHarmonicsL2Converter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using UnityEngine.Rendering;
+
+namespace Newtonsoft.Json.UnityConverters.Math
+{
+    public class SphericalHarmonicsL2Converter : PartialFloatConverter<SphericalHarmonicsL2>
+    {
+        private const int COEFFICIENT_COUNT = 9;
```

Comment:
Should add a note where these values comes from, like how it's done in Src/Newtonsoft.Json.UnityConverters/AI/NavMesh/NavMeshQueryFilterConverter.cs